frontend/packages/vkt/src/components/publicEnrollment/PublicEnrollmentGrid.tsx

```diff
@@ -124,7 +124,8 @@ export const PublicEnrollmentGrid = ({
     (isAuthenticatePassed && !reservation);
 
   const isShiftedFromQueue =
-    enrollment.status === EnrollmentStatus.SHIFTED_FROM_QUEUE;
+    enrollment.status === EnrollmentStatus.SHIFTED_FROM_QUEUE ||
```

Comment:
Eiks näillä jonosta nostetuilla ole aina payment link? Tai näin toteutusta testaamatta mietin muuttaako tä muutos stepperin näkyvyyttä jotenkin?